aws-encryption-sdk-java.iml

```diff
@@ -0,0 +1,215 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We probably don't want to commit this?

Reply:
Not sure where this file came from, but if this is an IDE/build artifact, let's also add it to the `.gitignore`.